tools/src/main/scala/scala/scalanative/codegen/Generate.scala

```diff
@@ -115,6 +115,31 @@ object Generate {
       )
     }
 
+    def genClassInitializers(): Unit = {
```

Comment:
Why was this extracted here? It doesn't seem to have anything to do with the encoding of `j.l.Class`es.

Reply:
In one of previous versions I moved here both Ergys class initializers for Reflect support as well as my j.l.Class initializers, which were later removed, however I've left the method. Right now it does nothing to do with j.l.Classes although it makes code a bit more clear so I thought it may stay here.

Reply:
OK, that makes sense, but then this change must be on a separate PR. Otherwise it's going to get lumped together in the squashed commit for this PR, and the next person looking at that commit during a git blame will get super confused.

Reply:
I've reverted this change